boot/zephyr/prj.conf

```diff
@@ -12,6 +12,7 @@ CONFIG_MBEDTLS_CFG_FILE="config-boot.h"
 CONFIG_HEAP_MEM_POOL_SIZE=16384
 
 CONFIG_FLASH=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
```

Comment:
A bit dirty :), but that makes sense.

Hopefully your patch doesn't get bikeshedded too much longer.